src/components/CreateMenu.js

```diff
@@ -59,7 +59,7 @@ const CreateMenu = (props) => {
                     ])}
                 >
                     <View style={styles.createMenuIcon}>
-                        <Icon src={icon} width={variables.iconSizeLarge} height={variables.iconSizeLarge} />
+                        <Icon src={icon} />
```

Comment:
Just out of curiosity - when is it best to include a width/height, and when is it best to not include it? I see below there are places where we don't include it (maybe because the default is what we need?)

Reply:
The Icon component definition _(`src/components/Icon/index.js`)_ sets the icon's height and width to `variables.iconSizeNormal` by default. Because of that, we just really need to include it when we want a different icon size.
Because with this PR we would just have `iconSizeNormal` and `iconSizeSmall` available, we would only need it when using `iconSizeSmall`.